### ISSUE TITLE
mnist dataset 만들기

### ISSUE THREAD

**Issue by sgkim126** [cloudtrack/dsstne-starter#4]

**Comment by norangLemon** [comment]
dsstne 에서 사용하는 데이터포맷에 대한 설명 : https://github.com/amznlabs/amazon-dsstne/blob/master/docs/getting_started/userguide.md

**Comment by norangLemon** [comment]
mnist 데이터 페이지 : http://yann.lecun.com/exdb/mnist/

**Comment by norangLemon** [comment]
위 링크들을 참고하여 amazon dsstne 에서 읽을 수 있는 포맷으로 mnist 데이터를 변환하는 파이썬 스크립트를 만들어야 함. (ntopia comments)

**Comment by norangLemon** [comment]
현재 파싱 코드 작성을 완료하였으며,
training set과 test set parsing 결과물도 추가되었습니다.

결과 파일 및 코드는 [여기](https://github.com/norangLemon/mnistForDsstne)에서 볼 수 있습니다.
